src/App.js

```diff
@@ -1,25 +1,42 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from "react";
+import Header from "./components/Header";
+import Footer from "./components/Footer";
+// import Footer3 from "components/footers/Footer3.js";
```

Comment:
Line 4 can likely be removed.

Reply:
Yes, thank you. We were experimenting with something and forgot to remove it!